kategory-effects-rx2/src/main/kotlin/kategory/effects/data/FlowableKW.kt

```diff
@@ -1,13 +1,14 @@
-package kategory
+package kategory.effects
 
+import kategory.*
 import io.reactivex.*
 
 fun <A> Flowable<A>.k(): FlowableKW<A> = FlowableKW(this)
 
 fun <A> FlowableKWKind<A>.value(): Flowable<A> = this.ev().flowable
 
 @higherkind
-@deriving(Functor::class, Applicative::class)
+@deriving(Functor::class, Applicative::class, Monad::class)
```

Comment:
Are these Foldable and Traversable? I'm assuming that if they have an empty value others could follow.

Reply:
Does Fold/Traverse require termination? If so, then they aren't as termination isn't assured.

Reply:
SequenceKW can model infinite streams and has a Foldable and Traverse instance